### ISSUE TITLE
Sub-navigation

### ISSUE THREAD

**Issue by marhigh** [stefanjudis/perf-tooling#436]
Would it be nicer if the site had sub-navigation to more easily introduce content categories to visitors?

**Comment by stefanjudis** [comment]
@marcobiedermann 

I think that's a good idea. We could count the most used categories per entry automatically and then display the top5 or s.th.?

What do you think?

**Comment by marcobiedermann** [comment]
I like the idea, but I think a sub navigation if not the best solution for a friendly UX.
How about a tag cloud or some options to sort / filter is a sidebar on the right?
We already have an issue about filtering and sorting.
Can we link these issue topics @stefanjudis?

**Comment by stefanjudis** [comment]
A tag cloud? I'd like to give a veto for that. ;)

Sorting and filtering makes sense though.
Sure let's evaluate on filters then.

@marhigh would this fit your needs?

@marcobiedermann 

Do you maybe want to dribble around to find a design solution?
I'd close the other issue then and let's discuss over here. :bowtie:

**Comment by marhigh** [comment]
Sorry for the slow response, I think some options to sort / filter would definitely help!

**Comment by marcobiedermann** [comment]
I thought about some possible sorting and filtering options. What do you guys think about them.
Did I miss something. It would be great if we could sort by stars but I guess this is not possible because of the different platforms. More platforms does not mean more stars because they will add up.

### Sort
* A-Z  
* Z-A
* last updated (?) 

### Filter
#### Price
* Free  
* Paid

#### Workflow
* Grunt
* Gulp
* CLI

#### Language
* Node
* PHP
* JavaScript

#### Framework
* Angular
* Wordpress

#### Browser
* Chrome
* Firefox
* ~~Safari~~
* ~~IE~~

#### OS & Apps
* Mac
* Windows
* Linux
* Illustrator

**Comment by stefanjudis** [comment]
For simplicity I'd put all the platforms together as we do it with the colorful bubbles.

:+1: For sorting
:+1: paid/unpaid

**Comment by marcobiedermann** [comment]
@stefanjudis Yes, the filter will include all platforms. I just grouped them the separate them visually.
Or do you prefer one big list?

**Comment by marcobiedermann** [comment]
I thought of something similar to this.

All together:
![perf-tooling-all](https://cloud.githubusercontent.com/assets/5244986/6674726/05529ac6-cc1c-11e4-878f-811e4b824222.png)

Grouped:
![perf-tooling-grouped](https://cloud.githubusercontent.com/assets/5244986/6674727/0552efda-cc1c-11e4-842e-5f29daf4e6c9.png)

**Comment by stefanjudis** [comment]
For me it'd be.

- Sort
- Price
- Platform
- Category

If case we want to split the platforms, I'd do this in a second step. Because this needs to be configurable and is a bigger overall change. Putting it together might be easier to start and decrease risk. :)

**Comment by marcobiedermann** [comment]
Is this how you imaging the sorting / filter should be like?

![perf-tooling-filter](https://cloud.githubusercontent.com/assets/5244986/6675434/a023bcac-cc20-11e4-9b0d-4cdaa675c7d6.png)

**Comment by stefanjudis** [comment]
I imagine category also as checkboxes but except that - pretty much yes. :)

**Comment by marcobiedermann** [comment]
All categories as checkboxes? There are a lot of them. I can try it. Let's see how it looks

**Comment by stefanjudis** [comment]
Then we have to clean them up. ;) We're having too much anyways. :laughing:

**Comment by marhigh** [comment]
sweet, guys! it's looking great!
how about something to do with level of difficulty? some guidance for us
n00bs?

**Comment by marhigh** [comment]
BTW small typo -> 'Last Updated' should be capitalized like the rest.

**Comment by stefanjudis** [comment]
@marhigh 

> how about something to do with level of difficulty? some guidance for us n00bs?

let's go step by step. :)

This feature will be a lot of work with tons of changes.
We have already this #228 - if this fits your needs we can tackle it next.
